datafusion/core/tests/sql/aggregates.rs

```diff
@@ -20,24 +20,6 @@ use datafusion::scalar::ScalarValue;
 use datafusion::test_util::scan_empty;
 use datafusion_common::cast::as_float64_array;
 
-#[tokio::test]
-async fn csv_query_avg_multi_batch() -> Result<()> {
```

Comment:
Here is the corresponding test in .slt

https://github.com/apache/arrow-datafusion/blob/dcd52ee3d87c4dd9e2c176165e9e20644f66988b/datafusion/core/tests/sqllogictests/test_files/aggregate.slt#L21-L25

👍